### PR TITLE
Initial SQL

### PR DESCRIPTION
**Pull request by kevinelliott**
This is a draft PR intentionally so that we can discuss and shape this initial SQL pass.

### CODE REVIEW

File: postgres/sql/meshinfo.sql [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  created_at timestamp with time zone DEFAULT now() NOT NULL
+);
+
+CREATE TABLE node_positions (
+  id bigint NOT NULL PRIMARY KEY,
+  node_id bigint NOT NULL,
+  latitude double precision NOT NULL,
+  longitude double precision NOT NULL,
+  altitude double precision,
+  geom public.geometry(PointZ,4326),
```
Comment (ab0oo) [inline postgres/sql/meshinfo.sql:24]: If we're going to leave in the 'geom' columns, we're going to need to ensure the postgis extensions are loaded, either ad DB setup or at some point later.

**Comment by ab0oo** [comment]
We need to ensure the indices are created, as well.  I think there are some table joins in the views that require indices or they just explode.  Also, there's at least one sequence in there, but it might be autogenerated at table create time.
We also need to make sure table ownership is correct.  I guess that can be handled by the create script when the DB is instantiated.

File: postgres/sql/meshinfo.sql (hunk 1 above)
Comment (kevinelliott) [reply postgres/sql/meshinfo.sql:24]: Good point, let me add this to the top. Any thoughts on other extensions we are missing?

**Comment by kevinelliott** [comment]
> We need to ensure the indices are created, as well. I think there are some table joins in the views that require indices or they just explode. Also, there's at least one sequence in there, but it might be autogenerated at table create time. We also need to make sure table ownership is correct. I guess that can be handled by the create script when the DB is instantiated.

Indeed, indexes. Feel free to take a pass here and push up a commit if you have a moment. Otherwise, I can tackle. I was mostly waiting until we solidified the structure.

**Comment by kevinelliott** [comment]
@ab0oo Of course this PR is based on a logical "merging" of each of our original schemas, with the intention to provide similar capabilities while also preserving history data. I would think we'll need a few more tables and perhaps a couple more views.

**Comment by daviesgeek** [comment]
This looks good to me, i'm lacking a lot of context of the data structures though, so take my input with a grain of salt.

**Comment by kevinelliott** [comment]
Merging as initial SQL but we can adjust and add to in follow ups.

**Pull request merged**